### DISCUSSION TITLE
[Question] 控件虚拟化是什么

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#5108] [提问] [answered]
不太懂 WPF 控件 awa
想知道这个是什么意思

**Comment by MoYuan-CN** [comment]
大概是和 Vue 的 V-Dom（虚拟 Dom） 差不多的东西

**Reply by wuliaodexiaoluo** [reply]
>大概是和 Vue 的 V-Dom（虚拟 Dom） 差不多的东西

看不懂，但是感觉很高级.....

**Comment by wuliaodexiaoluo** [comment, marked as answer]
浅看了眼微软官方文档
控件虚拟化会在控件可见时加载数据
相当于优化了加载速度（
https://learn.microsoft.com/zh-cn/dotnet/desktop/wpf/advanced/optimizing-performance-controls?view=netframeworkdesktop-4.8
